### ISSUE TITLE
Accessibility focus message

### ISSUE THREAD

**Issue by decabeza** [consul/consul#1850]
Background
==

When users not logged try to support a proposal appears a message to sign in or sign up to continue.

The problem is if **the user navigate with the keyboard using tab key**, the message appears on _focus_ but when _blur_ this message keep showing, so... the user can't view the supports info.

Screenshot
==
<img width="901" alt="participation not allowed" src="https://user-images.githubusercontent.com/631897/30284598-a6d89d6a-971b-11e7-9273-d0c30387ab70.png">

What we need
==

The message has to appear on `mouseover/focus` with tab, allow navigate on `sign in/ sign up links` and hide again on `mouseleave/blur`.

The desired behaviour is: 😌 
(The user should be navigate over the links with the keyboard and after tab over a proposal and tab over next one keep seeing the number supports info)

![tabs_desired](https://user-images.githubusercontent.com/631897/30491493-c2a592d0-9a3d-11e7-92f2-78efbe61cdf0.jpg)

**Comment by aitbw** [comment]
I'll take this one.

**Comment by decabeza** [comment]
Start with [this PR](https://github.com/consul/consul/pull/1877), but although the message disappear 👍  the user can't tab over "sign in/sign up" links 😞

With this changes the behaviour is:

![tabs_now](https://user-images.githubusercontent.com/631897/30491484-b4cf4c00-9a3d-11e7-8e6b-2f71c50a5461.jpg)

But the desired behaviour is: 😌 
(The user should be navigate over the links with the keyboard and after tab over a proposal and tab over next one keep seeing the number supports info)

![tabs_desired](https://user-images.githubusercontent.com/631897/30491493-c2a592d0-9a3d-11e7-92f2-78efbe61cdf0.jpg)

**Comment by decabeza** [comment]
Maybe this can be a solution for this issue 😃  
https://css-tricks.com/keeping-parent-visible-child-focus/
